### PR TITLE
Fix security vulnerabilities

### PR DESCRIPTION
**Pull request by SVarallo**
Jira ticket at: https://companieshouse.atlassian.net/browse/LFA-1721

As in https://github.com/companieshouse/promise-to-file-web/pull/116 there is one vulnerability left outstanding for yargs-parser in mockserver but there is currently no fix for this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,16 +49,17 @@
...
     "nodemon": "^1.18.9",
     "sonarqube-scanner": "^2.6.0",
     "supertest": "^3.4.2",
-    "ts-jest": "^24.1.0",
+    "ts-jest": "^25.5.1",
     "ts-node": "^8.0.2",
     "tslint": "^5.13.0",
-    "typescript": "^3.3.1"
+    "typescript": "^3.3.1",
+    "yargs-parser": "^18.1.3"
```
Comment (markpit) [inline package.json:63]: you've got a new package here, did this get added from npm audit fix?

Comment (markpit) [reply package.json:63]: yargs-parser

Comment (SVarallo) [reply package.json:63]: No I added it as a fix to one of the vulnerabilities I was getting

Comment (SVarallo) [reply package.json:63]: Ah nevermind, running an audit without it produces the same result, I'll revert it.

Comment (mattch1) [reply package.json:63]: Yes you won't need it.

**Pull request merged**